menu_konsolowe.py

```diff
@@ -0,0 +1,95 @@
+
+
+
+# pozostałość logowania - do rewizji
+# def admin_account():
+#     adm_login = input("Podaj login:")
+#     adm_password = input("Podaj Hasło:")
+
+
+def main_menu():
+
+    print("1. ZALOGUJ SIĘ JAKO KLIENT")
+    print("2. ZALOGUJ SIĘ JAKO ADMINISTRATOR")
+    print("---------------------------------- \n")
+    user_input = input("Wybierz panel do którego chcesz się zalogować: ")
+    print("---------------------------------- \n")
+    if user_input == "1":
+        print("Zalogowałeś się do panelu klienta")
+
```

Comment:
nit: white space https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak